pkg/auth/handler/me.go

```diff
@@ -57,6 +64,26 @@ func (h MeHandler) DecodeRequest(request *http.Request) (payload handler.Request
 }
 
 func (h MeHandler) Handle(req interface{}) (resp interface{}, err error) {
-	resp = h.AuthContext.AuthInfo()
+	authInfo := h.AuthContext.AuthInfo()
+
+	// refresh access token with a newly generated one
```

Comment:
Please move this inline comment to descriptive comment at Handler.

Also, please add example input and output.